src/HeaderRow.tsx

```diff
@@ -90,9 +84,10 @@ function HeaderRow<R, SR, K extends React.Key>({
     <div
       role="row"
       aria-rowindex={1} // aria-rowindex is 1 based
-      ref={ref}
-      tabIndex={tabIndex}
-      className={clsx(headerRowClassname, className)}
+      className={clsx(headerRowClassname, {
+        [rowSelectedClassname]: selectedCellIdx === -1
```

Comment:
Is it possible to select the whole header row?

Reply:
![image](https://user-images.githubusercontent.com/1452717/150815913-3fb3316d-22f0-4e0b-b0a1-97c212070308.png)

If up arrow is pressed on the first grouped row